### DISCUSSION TITLE
Issue with field of type pattern not appearing in email template

### DISCUSSION THREAD

**Discussion by romainpoirier** [solspace/craft-freeform#966] [Questions & Answers] [answered]
**Describe the bug or issue you're experiencing**

I encountered a problem with a field of type Pattern in my Admin email template.

**Steps to reproduce**

1. Set up Craft CMS with the mentioned versions.
2. Create a Freeform form that includes a field of type Pattern with the following value:
```
^[0]*[0-9]{2}[.\- ]{0,1}[0-9]{2}[.\- ]{0,1}[0-9]{2}[.\- ]{0,1}[0-9]{3}[.\- ]{0,1}[0-9]{2}$
```
3. Configure an email notification in Freeform using the provided Admin email template below.

**Admin Email Template Example**:
```twig
{%
	for field in submission.fieldMetadata
	if field.type not in ['dynamic_recipients', 'hidden', 'multiple_select', 'rating', 'regex']
%}
	{% if field.type == 'file' %}
		{% set assetIds = attribute(submission, field.handle).value ?? [] %}
		{% if assetIds | length %}
			{%
				for assetId in assetIds
				if craft.assets.id(assetId).one() | length
			%}
				{% set asset = craft.assets.id(assetId).one() %}
				<a href="{{ asset.url }}" style="color:#000;text-decoration:none;">
					{{ asset.filename | typogrify }}
				</a>
			{% endfor %}
		{% else %}
			{{ 'Blank' | t | typogrify }}
		{% endif %}
	{% else %}
		{{ '{value}' | t({ value: (field.valueAsString is not empty) ? field.valueAsString : 'Blank' | t }) | typogrify }}
	{% endif %}
{% endfor %}
```

**User-Side Template Example**:
```twig
{% for field in allFields %}
	{{ '{value}' | t({ value: (field.valueAsString is not empty) ? field.valueAsString : 'None' | t }) | typogrify }}
{% endfor %}
```

4. Submit a form with data that should trigger the display of the field of type Pattern in the email.

**Expected behavior**

The field of type Pattern should appear in the Admin email template as it did before.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 3.7.52
- Freeform Version: 3.13.18

**Additional context**

I have verified that the field of type Pattern is not appearing in the Admin email template, and using `allFields` as suggested in the user-side template does not resolve the issue.

**Comment by romainpoirier** [comment]
Update: after removing the pattern value I can confirm that the field is still not present in the email notification, so maybe it's an issue with _Pattern_ field.

**Comment by kjmartens** [comment, marked as answer]
Hi @romainpoirier. There is no `Pattern` field type... I assume you mean `Regex`? Perhaps you have a translation file that is renaming it as `Pattern`, but that is only visual. The handle for the field type is still `regex`. 🙂 

Also, your conditional is preventing `regex` fields from showing in the email notification:

``` twig
if field.type not in ['dynamic_recipients', 'hidden', 'multiple_select', 'rating', 'regex']
```
